flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/TiKVRichParallelSourceFunction.java

```diff
@@ -114,6 +119,9 @@ public void open(final Configuration config) throws Exception {
         cdcClient = new CDCClient(session, keyRange);
         prewrites = new TreeMap<>();
         commits = new TreeMap<>();
+        // if pull cdc event block when region split, cdc event will lose.
+        // use queue to make sure pull event unblock.
+        committedEvents = new LinkedBlockingQueue<Cdcpb.Event.Row>(50000000);
```

Comment:
Capacity `50000000` can you give me some explain?

Reply:
done